guideline-02.md

```diff
@@ -1,5 +1,4 @@
-## Do Not Advertise or Promote Products
+## Don't spam
```

Comment:
I would change this to "Do Not Spam"

When possible, it's recommended to not use contractions (helps in translations)